pi_graphreport/class.tx_caretaker_pi_graphreport.php

```diff
@@ -184,6 +188,15 @@ public function getNodes()
 
         return $nodes;
     }
+
+    private function initDirectories()
+    {
+        if (!is_dir(PATH_site . self::PATH_CHARTS)) {
+            if (!mkdir(PATH_site . self::PATH_CHARTS, 0770, true)) {
```

Comment:
This prevents the TYPO3 API handling (user and permission configuration) for new files and folders.